cmd/osm-controller/validate.go

```diff
@@ -32,42 +34,46 @@ func validateCLIParams() error {
 		return errors.Errorf("Invalid --webhook-config-name value: '%s'", webhookConfigName)
 	}
 
+	if caBundleSecretName == "" {
+		return errors.Errorf("Please specify the CA bundle secret name using --ca-bundle-secret-name")
+	}
+
 	return nil
 }
 
 func validateCertificateManagerOptions() error {
-	switch *osmCertificateManagerKind {
-	case tresorKind:
+	switch osmCertificateManagerKind {
```

Comment:
Does it make sense to move this validation code in the new `certproviderutil` ?

Reply:
Done.